groups/ecorp/local.json

```diff
@@ -1,15 +1,27 @@
 {
-  "comment":"ECorp evil AS",
+  "comment": "ECorp evil AS",
   "containers": {
-    "infra":
-        {"container":"infra",
-          "interfaces":[
-            {"bridge":"lan", "ipv4":"100.81.0.2/16", "ipv6":"2001:db8:81::2/48"}
-          ],
-          "gatewayv4":"100.81.0.1",
-          "gatewayv6":"2001:db8:81::1",
-          "templates":[
-            {"template":"updatecaroots"},
-            {"template":"nodhcp", "domain":"ecorp.milxc", "ns":"100.100.100.100"}]}
```

Comment:
As we discussed out of bands (twitter), I think we should keep json files dense. Dense, coherent and clean. Perhaps we should treat them in a dedicated PR ?

Reply:
The problem with this level of json compression is that it must be done by hand. It would be better to compress it as much as possible even if it means beautifying it before modifying it in order to be able to use a tool. What do you think?